examples/todo/rust/main.rs

```diff
@@ -96,3 +96,15 @@ pub fn main() {
 
     main_window.run().unwrap();
 }
+
+#[cfg(target_os = "android")]
+#[no_mangle]
+fn android_main(app: i_slint_backend_android_activity::AndroidApp) {
+    slint::platform::set_platform(Box::new(
+        i_slint_backend_android_activity::AndroidPlatform::new_with_event_listener(app, |event| {
+            eprintln!("Got event: {event:?}")
```

Comment:
I suggest to remove that :)

Reply:
This was just as the example.  Do you think it's bad to have?

Reply:
For an example, I think it's weird yes :)